lib/src/detectors.js

```diff
@@ -200,6 +200,63 @@ const addDetectorApi = (matroid) => {
       this._genericRequest(options, resolve, reject);
     });
   };
+
+  matroid.addFeedbackFromFile = function(detectorId, imageFile, feedback) {
+    /*
+     * Add feedback to a Matroid detector from a local file.
+     */
+    return new Promise((resolve, reject) => {
+      const feedbackToAdd = Array.isArray(feedback) ? feedback : [feedback]
+      
+      const options = {
+        action: 'addFeedbackFromFile',
+        uriParams: { ':detectorId': detectorId },
+        filePaths: imageFile,
+        data: {
+          feedback: feedbackToAdd.map(item => JSON.stringify(item)),
+        }
+      };
+
+      this._genericRequest(options, resolve, reject);
+    });
+  }
+
+  matroid.addFeedbackFromURL = function(detectorId, url, feedback) {
+    /*
+     * Add feedback to a Matroid detector a url
+     */
+    return new Promise((resolve, reject) => {
+      const options = {
+        action: 'addFeedbackFromURL',
+        uriParams: { ':detectorId': detectorId },
+        data: {
+          feedback: Array.isArray(feedback) ? feedback : [feedback],
+          url,
+        }
+      };
+
+      this._genericRequest(options, resolve, reject);
+    });
+  }
```

Comment:
It seems like elsewhere in the API (e.g. `classifyImage` in `images.js`), we just have one method for routes that can accept files or urls. Instead of having separate methods, we expect the image to have the format `{ url: example.com }` or `{ file: '/example/path' }` and then check for the presence of either the `url` or `file` key and handle accordingly. I'll defer to you as to whether that would be preferred here.

Reply:
Yeah, so I had thought this was a bit more clear, but the other way would be more consistent with what we do elsewhere (and matches what Clarifai does in their API fwiw). I think I'll make the update.